theme/js/components/search/suggest-results.vue

```diff
@@ -25,9 +25,7 @@ Used by the suggest feature to display typeahead-style results when you type you
         <span v-html="arrow" />
       </a>
       <p class="text-grey-300 m-0">
-        Nullam neque bibendum convallis enim aliquam. Integer egestas accumsan,
-        in varius lectus. Elementum a facilisis nibh pellentesque enim egestas
-        porta.
+        {{ $t('Datasets are collections of data, i.e. structured information, easily readable by a machine.') }}
```

Comment:
Pas trop self-explanatory. Je suggère "Datasets are a collection of one or more resources, i.e. data files that can be easily read by a machine"